src/queueing_honey_badger/mod.rs

```diff
@@ -101,9 +101,11 @@ where
     }
 
     /// Creates a new Queueing Honey Badger instance with an empty buffer.
-    pub fn build<R: Rng>(self, rng: &mut R) -> QueueingHoneyBadgerWithStep<T, N, Q> {
+    pub fn build<R: Rng>(self, rng: &mut R) -> Result<QueueingHoneyBadgerWithStep<T, N, Q>>
+    where
+        R: 'static + Rng + Send + Sync,
```

Comment:
This `where` clause is probably not necessary, right (looks like a merge-artifact). The `Rng` trait bound is already present, and `'static + Send + Sync` have been carefully removed.

Reply:
I amended the commit. (Then I force-pushed _again_ to rebase.)